Soleras3/react-challenge-project-jan-2023
# Fix order submit

## Changes
- Fix order select onChange events to store expected values.

## Purpose
Orders placed were not added to the orders list. See: https://github.com/Shift3/react-challenge-project-jan-2023/issues/1

## Approach
Select onChange's `event.value` is undefined and JS doesn't produce an error for accessing it. The value is in `event.target.value`.

## Testing Steps
1. Log in then navigate to Order Form
2. Select an order option and a quantity value
3. Submit the order
4. Navigate to Order List and observe the new order was created

## Learning
Pretty simple dev error since JS doesn't outright complain about accessing undefined properties.
Closes https://github.com/Shift3/react-challenge-project-jan-2023/issues/1
